### PR TITLE
[ADF-4948] Move the resources config in the process cloud lib - Part 2

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Right now all the resources config of differents apps are inside the same file

**What is the new behaviour?**
Single responsibility

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/resources/resources.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* cSpell:disable */
+export const ACTIVITI_CLOUD_APPS: any = {
```
Comment (cristinaj) [inline lib/process-services-cloud/src/lib/resources/resources.ts]: These hardcoded values are related to the env and applications we use. This shouldn't be part of testing-package.

File: e2e/process-services-cloud/apps-section-cloud.e2e.ts [hunk 2]
```diff
@@ -33,7 +33,10 @@ describe('Applications list', () => {
     const navigationBarPage = new NavigationBarPage();
     const settingsPage = new SettingsPage();
     const appListCloudPage = new AppListCloudPage();
-    const simpleApp = browser.params.resources.ACTIVITI7_APPS.SIMPLE_APP.name;
+    const simpleApp = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.name;
+    // tslint:disable-next-line:no-console
+    console.log('Stampo' + simpleApp);
```
Comment (adinapitul) [inline e2e/process-services-cloud/apps-section-cloud.e2e.ts]: is this Italian? :)

Comment (mauriziovitale) [reply e2e/process-services-cloud/apps-section-cloud.e2e.ts]: ahaha
yep just playing with the comment

File: protractor.conf.ts [hunk 3]
```diff
@@ -16,6 +17,8 @@ const width = 1366, height = 768;
 let ENV_FILE = process.env.ENV_FILE;
 let GROUP_SUFFIX = process.env.PREFIX;
 
+RESOURCES.ACTIVITI_CLOUD_APPS = RESOURCES_CLOUD.ACTIVITI_CLOUD_APPS;
```
[the remark this answers is not in the document]
Comment (mauriziovitale) [reply protractor.conf.ts]: We were already loading the app setting from
`const RESOURCES = require('./e2e/util/resources');`
I have just moved the cloud setting as part of the `adf-testing/cloud` folder

File: .travis.yml [hunk 4]
```diff
@@ -110,6 +110,7 @@ jobs:
           script: ./scripts/travis/e2e/process-services-cloud-e2e.sh
 
 
+
```
Comment (eromano) [inline .travis.yml]: space not needed

File: lib/testing/src/lib/core/utils/browser-visibility.ts [hunk 5]
```diff
@@ -17,51 +17,50 @@
 
 import { browser, by, element, ElementFinder, protractor } from 'protractor';
 
-const until = protractor.ExpectedConditions;
```
[the remark this answers is not in the document]
Comment (mauriziovitale) [reply lib/testing/src/lib/core/utils/browser-visibility.ts]: It was giving a compilation error something like
ExpectedConditions of undefined

File: protractor.conf.ts [hunk 6]
```diff
@@ -5,6 +5,8 @@ const tsConfig = require('./e2e/tsconfig.e2e.json');
 const AlfrescoApi = require('@alfresco/js-api').AlfrescoApiCompatibility;
 const TestConfig = require('./e2e/test.config');
 const RESOURCES = require('./e2e/util/resources');
+const RESOURCES_CLOUD = require('@alfresco/adf-testing');
```
[the remark this answers is not in the document]
Comment (mauriziovitale) [reply protractor.conf.ts]: Yep for now we need to build the testing project in case you change something in the resources.

**Pull request merged**